src/main/scala/workflow/DelegatingTransformer.scala

```diff
@@ -0,0 +1,9 @@
+package workflow
+
+import org.apache.spark.rdd.RDD
+
+class DelegatingTransformer[T](override val label: String) extends TransformerNode[T] {
```

Comment:
whoa, this is scary sounding! What does it do? Can we have some comments?

Should this be package protected?